less/mixins.less

```diff
@@ -1 +1,23 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.home button {
+    display: block;
+    background-color: #17A2B8;
+    margin: 0 10px 20px;
+    width: 175px;
+    height: 40px;
+    font-size: 2rem;
+    border-radius: 10px;
+    color: white;
+
+    @media @mobile{
+      display: block;
+      background-color: #17A2B8;
+      margin: 0 10px 20px;
+      width: 300px;
+      height: 40px;
+      font-size: 2rem;
+      border-radius: 10px;
+      color: white;
+    }
+  }
```

Comment:
_looks deeply suspicious_ 🤔 

That doesn't look much like a mixin to me... in the mixins file!